src/App.jsx

```diff
@@ -4,29 +4,54 @@ import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
+import { getTodos } from './api';
+
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
   };
 
+  async componentDidMount() {
+    const todos = await getTodos();
+
+    this.setState({
+      todos,
+    });
+  }
+
+  selectUser = (selectedUserId) => {
+    this.setState({ selectedUserId });
+  }
+
   render() {
     const { todos, selectedUserId } = this.state;
 
-    return (
+    return todos.length !== 0 ? (
       <div className="App">
         <div className="App__sidebar">
-          <TodoList todos={todos} />
+          <TodoList
+            todos={todos.data}
```

Comment:
Better take data in your API file, because if server system will change you must solve mistake in a lot of places
```suggestion
            todos={todos.data}
```

Reply:
pass from getTodos todos.data, so you won't need to use in component